src/modules/developer-guide/pages/documentation/Writing-guidelines1.adoc

```diff
@@ -0,0 +1,113 @@
+include::ROOT:partial$variables.adoc[]
+
+[#documentation-writing-guidelines]
+= Writing Guidelines
+
+Here, we outline the base guidelines that you should to follow when contributing to the documentation.
+
+You can refer to https://docs.openstack.org/doc-contrib-guide/writing-style/general-writing-guidelines.html[this article] or any other article for additional insight.
+
+== Use active voice
+
+Active voice *identifies the subject* that has performed the given action; for example:
+
+> To configure external login providers go into the administration panel, authentication and configure ...
+
+This style makes the *reader more involved* and therefore makes it friendlier and *more comfortable to read*.
+It also makes it a bit *shorter*.
+
+There are *some exceptions* to using the active voice.
+
+Use the passive voice:
+
+* When we are *talk about errors*; the active voice can suggest blaming the user.
+* When it *shortenes* the message.
+* When *the subject isn't and doesn't need to be recognised*.
+This includes sentences in which you wish to identify the first person.
+** This often overlaps with the use of tense.
```

Comment:
Can you restructure like so, please?

```
.Use the passive voice:
* When we are...
...
```
The `.text here...:` makes "the thing belong" to the list